.gitignore

```diff
@@ -36,3 +36,4 @@ man/man5
 man/man8
 pyenv
 vendor/pkg/
+.docker
```

Comment:
Hmm, so why not create a global `gitignore` or just `.git/info/exclude` in the cloned repo? (https://help.github.com/articles/ignoring-files/#create-a-global-gitignore)

I feel like this is a pretty strong edge case that most people won't be hitting, so it feels strange to put something so environmental in the repo-managed `.gitignore`.